### ISSUE TITLE
Zsh completion not work as oh-my-zsh plugin

### ISSUE THREAD

**Issue by snowman** [cheat/cheat#616]
**A description of the problem**
Zsh completion as oh-my-zsh plugin shows error when start terminal.

**cheat version info**
4.2.0

**cheat configuration info**
steps to reproduce

```sh
$ mkdir -p ~/.oh-my-zsh/custom/plugins/cheat/
$ wget https://raw.githubusercontent.com/cheat/cheat/master/scripts/cheat.zsh -O ~/.oh-my-zsh/custom/plugins/cheat/cheat.plugin.zsh

$ cat ~/.zshrc
export ZSH=$HOME/.oh-my-zsh
ZSH_THEME="robbyrussell"
plugins=(cheat)
source $ZSH/oh-my-zsh.sh
```

When restart terminal, it shows error:

    _arguments:comparguments:325: can only be called from completion function

**Comment by chrisallenlane** [comment]
Hi, @snowman

Thanks for the bug report.

Unfortunately, I personally have no experience with `zsh` or `oh-my-zsh`. That being the case, I tagged this issue as `help-wanted` and pinned it for visibility. Hopefully someone within the community will be able to help you out.

**Comment by PabloLec** [comment]
I was facing the same issue.
I don't know if the current config did work at some point, I've been familiar with zsh, oh my zsh and their plugins for about a year. Maybe a previous update broke it?

Anyway, current config could not work properly, I added #629 fixing it.

**Comment by chrisallenlane** [comment]
@PabloLec, thanks very much! Your patch is included in release `4.2.2`.
